helm-charts/strimzi-kafka-operator/values.yaml

```diff
@@ -1,5 +1,8 @@
 # Default values for strimzi-kafka-operator.
 
+strimzi:
+  watch_namespaces: []
```

Comment:
Will an empty list mean the cluster operator doesn't watch any namespace?  I notice in `_helpers.tpl` you have a definition that checks to see if this option is defined, if not use the `.Release.Namespace` which should be the default option IMO.

Reply:
an empty list means the cluster operator only watches the default namespace or the one passed to helm with --namespace

check my comment on the _helpers.tpl for a detailed explanation :)

Reply:
Cool.  BTW, I would recommend putting `watch_namespaces` at the top level.  All configuration is Strimzi-related, so it seems redundant to have one configuration item under a `strimzi` top level item.

Reply:
Also, please use camelCase to remain consistent with the rest of the `values.yaml`.